__tests__/app.test.js

```diff
@@ -93,4 +93,42 @@ describe('GET: /api/articles', () => {
             })
         })
     });
+});
+
+describe('GET: /api/articles/:article_id/comments', () => {
+    it("200: responds with all comments relating to particular article with article_id", () => {
+        return request(app).get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body}) => {
+            const {comments} = body;
+            expect(comments.length).toBe(11)
+            expect(comments).toBeSortedBy('created_at', {descending: true})
+            console.log(comments)
+            comments.forEach((comment) => {
+                expect(comment).toHaveProperty('comment_id')
+                expect(comment).toHaveProperty('body')
+                expect(comment).toHaveProperty('votes')
+                expect(comment).toHaveProperty('author')
+                expect(comment).toHaveProperty('article_id')
+                expect(comment).toHaveProperty('created_at')
+            })
+        })
+    })
+    it("GET:404 sends an appropriate error message when given a valid but non-existent article_id", () => {
+        return request(app).get('/api/articles/999/comments')
+        .expect(404)
+        .then(({body}) => {
+            const {msg} = body
+            // console.log(body)
+            expect(msg).toBe('Not found')
+        })
+    })
+    it("GET:400 sends an appropriate error message when given an invalid article_id", () => {
+        return request(app).get('/api/articles/dog/comments')
+        .expect(400)
+        .then(({body}) => {
+            const {msg} = body;
+            expect(msg).toBe("Bad request")
+        })
+    })
```

Comment:
We could do with another test here for an article that exists but has no comments!   This shouldn't be an error.  Should return a 200, but with no content!